bits-and-rats/readme.md

```diff
@@ -0,0 +1,133 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/N3qmN6pYhi0?si=c4pdiu8JwCHJtGfa" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## 🎯 Learning Objectives
+
+```objectives
+- [ ] Transform physical locations to binary patterns
+```
+
+## 🎭 The Story
+
+{{<note title="A long awaited party" type="activity">}}
+**The King** has stocked his wine cellar with one thousand bottles of wine for his birthday party tomorrow. However, the King's evil brother, the Duke, has poisoned one of the bottles! The poison is lethal and will kill anyone who takes even one sip within one hour. You, the Butler, must find the poisoned bottle and save the party! But how?
+{{</note>}}
+
+## 🎩 The Players
+
+- 🤵🏻 **Butlers**: The players who will be planning the tests.
+- 🍾 **Bottles**: The players who will be tested.
+- 🐀 **Rats**: The players who will be testing the bottles.
+
+## 🎮 Setup
+
+> [!TIP]
+> How you play this game depends on how many people you have. If you have fewer than 12 people go to [Small Group Workshop](#small-group-version). If you have 12+ people, read on.
+
+You will need:
+
+- paper or sticky notes
+- pens
+- a space large enough to stand in formation
+
+Before you begin. **Facilitator**, make up a stack of pieces of paper with either 1 or 0 written on. You're going to hand each bottle player 8 pieces of paper. For 8 bottle players, you're going to need:
+
+- 7 x 8 pieces with 1
+- 1 x 8 pieces with 0
+
+The "zeroes" bottle is the poisoned bottle. You must choose one player to give the zeroes stack to. It's important that no players know which bottle is poisoned. Fold the paper over so nobody can see whose stack has the zeroes.
+
+Now assign the roles randomly and hand the bottle players their pieces of paper.
+
+### 🌱 Round 1: "The Simple Search" (4 Bottles)
+
+Four people will be bottles. Two people will be rats. Everybody else will be the butlers and come up with a testing strategy. This is your training round. Take your time to understand the game. Success is finding the poisoned bottle.
+
+Four (4) 🍾 **Bottles**, stand in any order the **🤵🏻 Butler** asks.
+
+Two (2) 🐀 **Rats**, follow the directions of your **🤵🏻 Butler/s**. For each round of testing you must go to the bottle or bottles directed and ask for a piece of paper.
+
+🍾 **Bottles**, when a 🐀 **Rat** comes to you, hand them one of your slips of paper. Don't peek!
+
+_When the **🤵🏻 Butler** yells 🧪 TEST..._
+
+🐀 **Rats** open all your paper slips. If you have a `0` on _any_ of your slips, please die as dramatically as you can manage.
```

Comment:
I feel like we should make clear "If the butlers run another round of testing, any dead rats can't be used"?